### PR TITLE
docs: fix http driver readme link

### PR DESCRIPTION
**Pull request by SergeyCherman**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [ X] The commit message follows our guidelines: CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[X ] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?

Currently the link http driver readme goes to a page not found.

Issue Number: N/A

## What is the new behavior?
Updated the http driver link in the main readme to point to the http driver readme in libs.

## Does this PR introduce a breaking change?

```
[ ] Yes
[X ] No
```

### CODE REVIEW

**Review by NachoVazquez — APPROVED** [review]
Thank you! And welcome to Lumberjack 😀

File: README.md [hunk 1]
```diff
@@ -493,7 +493,7 @@ export class AppModule {}
 
 ### HTTP driver
 
-For a more advanced log driver implementation, see [LumberjackHttpDriver](https://github.com/ngworker/lumberjack/blob/main/projects/ngworker/lumberjack/http-driver/README.md)
+For a more advanced log driver implementation, see [LumberjackHttpDriver](https://github.com/ngworker/lumberjack/blob/main/libs/ngworker/lumberjack/http-driver/README.md)
```
Comment (LayZeeDK) [inline README.md:496]: Let's add a comma to trigger another CI workflow run. A secret for SonarCloud was missing or expired. It should be fixed now.
```suggestion
For a more advanced log driver implementation, see [LumberjackHttpDriver](https://github.com/ngworker/lumberjack/blob/main/libs/ngworker/lumberjack/http-driver/README.md).
```

**Review by LayZeeDK — CHANGES_REQUESTED** [review]

**Comment by LayZeeDK** [comment]
@all-contributors add @SergeyCherman for docs

**Comment by LayZeeDK** [comment]
We're having some issues because of security protections. We need to modify our GitHub workflows to allow this PR. Please have patience a few days.

**Comment by SergeyCherman** [comment]
No worries, thanks for the fast review.

**Comment by NachoVazquez** [comment]
> No worries, thanks for the fast review.

We are facing some limitations with Github Actions. While we reach a solution, could you please create the PR from a clone of the original repository instead of your fork?

**Review by LayZeeDK — APPROVED** [review]

**Pull request closed without merging**